README.md

```diff
@@ -1,159 +1,6 @@
-# osm2lanes
+# Development
 
-This project transforms OpenStreetMap tags to a specification of lanes on a
```

Comment:
Too much got deleted here and not moved?

Reply:
oh, my move overwrote this, didn't notice :(